install-and-configure/install/multi-cluster/long-term-storage-configuration/long-term-storage-aws.md

```diff
@@ -40,7 +40,10 @@ config:
     enable: true
   part_size: 134217728
 ```
-> **Note:** Because this is a YAML, it requires this specific indention.
+
+{% hint style="info" %}
+Because this is a .yaml file, it requires the specific displayed indention.

```

Comment:
```suggestion
Because this is a YAML file, it requires the specific displayed indention.

```

Reply:
This hint really isn't necessary IMO because it's understood. That's literally how YAML works.

Reply:
Fair enough, we use YAML examples all over the docs without this disclosure, I think it was just written a long time ago